frontend/src/stores/usePreferencesStore.ts

```diff
@@ -20,7 +20,7 @@ const defaultPreferences: Preferences = {
   ctrlZoom: !navigator.platform?.match(/Win/) // Default to false on windows, which more often has a mouse
 }
 
-const usePreferencesStore = create<PreferencesStore>(persist((set: SetState<PreferencesStore>) => ({
+const usePreferencesStore = create<PreferencesStore>()(persist((set: SetState<PreferencesStore>) => ({
```

Comment:
Some of the creates need to have an extra function call like [described here](https://github.com/pmndrs/zustand/issues/1013#issuecomment-1203781863)